schema/meta.json

```diff
@@ -1,4 +1,7 @@
 {
+  "jsonRpcMethods": {
+    "cancel_request": "$/cancelRequest"
```

Comment:
Interesting question: I wonder if we need to have agent/client methods to handle this. 
Obviously not at the protocol level, but we do need to somehow route this to the agent/client 

Which means the agent/client will need to keep track of which request body was attached to each request id. If we require the cancellation to pass more params, we are close to just defining individual cancellation requests... 

🤔 interesting question. It might be fine to require on the SDK level that this is handled nicely for the implementer. But I think all the more reason to put this behind a capability for now, so we can also potentially roll this out initially as "unstable" and work out some of the wrinkles